dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -893,6 +894,13 @@ def SetUp(self,
         for f in flows:
           if f in client_flows:
             self.state.StoreContainer(containers.GrrFlow(host, f))
+            found_flows.append(f)
+
+    missing_flows = sorted([f for f in flows if f not in found_flows])
+    if len(missing_flows) != 0:
```

Comment:
```suggestion
    if missing_flows:
```

This is debatable, but I prefer it like this. Seems more pythonic.